docs/java-api/docs/index_.asciidoc

```diff
@@ -103,7 +103,7 @@ If you need to see the generated JSON content, you can use the
 
 [source,java]
 --------------------------------------------------
-String json = builder.string();
+String json = builder.getOutputStream().toString();
```

Comment:
org.elasticsearch.common.Strings.toString(builder) (or maybe better leave the full package name out)